examples/fuse_examples/tests/test_classification_stoic21.py

```diff
@@ -34,7 +34,7 @@
 class ClassificationStoic21TestCase(unittest.TestCase):
 
     def setUp(self):
-        self.root = tempfile.mkdtemp()
+        self.root = "tests2"#tempfile.mkdtemp()
```

Comment:
was this something temporary that you forgot to change back?

Reply:
temporary, thanks.